vuepress/docs/next/docs/releases/README.md

```diff
@@ -1,10 +1,197 @@
-# Entando 6.x Release Notes 
+
+# Entando 7.0.0 Release Notes
+
+## General
+
+Entando version 7.0 is a major release. It includes significant changes to the K8S infrastructure provisioning mechanism that provide better security, improved reliability and a simplified deployment process. Entando 7 introduces the [Entando Hub](../getting-started/landing-page.md#entando-hub), a rehauled compatibility matrix, development stack upgrades (e.g. JHipster 7.2) and several consolidation changes.
+
+## Supported Kubernetes Versions
+
+Explicitly supported versions:
+
+* GKE (K8s 1.20 - 1.21)
+* OKD4.8/OCP4.8 (K8s 1.21)
+* Vanilla K8s 1.21
+* K3s (K8s 1.21)
+
+## Security Updates
+
+* Fixed a missing escape in the CMS that could have led to XSS.
+
+* The visibility of bundle microservices has been restricted to only view their own Secrets.
+
+* Changed the vulnerable dependencies found in several modules to non-vulnerable versions: common-io, velocity, spring, guava, hibernate, struts2, taglibs, tika, apache-cxf, jackson, netty.
+
+    * Related list of CVEs: CVE-2021-22696, CVE-2021-27807, CVE-2021-27906, CVE-2021-28657, CVE-2017-6888, CVE-2021-20291, CVE-2020-17530, CVE-2020-10693, CVE-2015-5211, CVE-2020-25649, CVE-2018-1000873.
+
+* Addressed vulnerabilities: CVE-2021-44228, CVE-2022-21724, CVE-2021-22569, CVE-2022-23437, CVE-2022-23596, CVE-2021-25738, CWE-20, CWE-400, CWE-451, CVE-2022-21363.
+
+* Updated the ECG to generate projects with no vulnerable dependencies, with the exception of swagger-ui; a suitable patch for swagger-ui is not currently available. swagger-ui is activated in development mode and not enabled in the production maven profile.
+
+* Most of the existing ECG (Blueprint) vulnerabilities have been fixed; regardless, the user is strongly advised to rerun vulnerability scans every time a new bundle is generated. The remaining vulnerabilities will be fixed in the next minor version of Entando. These are not exploitable in production and only exposed during the internal development process. In particular, they concern:
+
+    * The dev webserver, used when developing the app
+    * The jest test files, used when running tests with jest
+    * General webpack configuration
+    * Dev utilities, used in the dev environment
+    * The PostCSS utility, used to import the CSS when building the final artifact
 
 ## New Features and Major Fixes
 
-### Entando Application
+### Entando Hub
+
+The Entando Hub is a new ECR feature that allows different EntandoApps to share bundles.
+* Hub Server: Allows creators to publish bundles that users can browse and clients can download.
+* Hub Client: Allows curators to browse, pull and install bundles from the Hub Server.
+
+### Application Engine
+
```

Comment:
@w-caffiero-entando the second statement is wrong, I believe. Only a manual process can be used to upgrade from 6.3.2 to 7.0 in order to be Liquibase compatible. I'd suggest removing the 2nd sentence. We have a separate ticket for adding notes on upgrades which can point to the 6.3.2->7.0.0 upgrade SQL script in the entando-de-app repo. What do you think?